.github/workflows/ecs-deploy.yml

```diff
@@ -0,0 +1,80 @@
+          echo "Getting task definitions"
+          ECS_TASK_DEFINITION_API=$(aws ecs describe-task-definition --task-definition brave-devices-${{ inputs.sdlc }}-api --region ${{ secrets.aws_region }} --query '{containerDefinitions: taskDefinition.containerDefinitions,family: taskDefinition.family,taskRoleArn: taskDefinition.taskRoleArn,executionRoleArn: taskDefinition.executionRoleArn,networkMode: taskDefinition.networkMode,volumes: taskDefinition.volumes,placementConstraints: taskDefinition.placementConstraints,requiresCompatibilities: taskDefinition.requiresCompatibilities,cpu: taskDefinition.cpu,memory: taskDefinition.memory}')
+          ECS_TASK_DEFINITION_API_MIGRATIONS=$(aws ecs describe-task-definition --task-definition brave-devices-${{ inputs.sdlc }}-api-db-migrations --region ${{ secrets.aws_region }} --query '{containerDefinitions: taskDefinition.containerDefinitions,family: taskDefinition.family,taskRoleArn: taskDefinition.taskRoleArn,executionRoleArn: taskDefinition.executionRoleArn,networkMode: taskDefinition.networkMode,volumes: taskDefinition.volumes,placementConstraints: taskDefinition.placementConstraints,requiresCompatibilities: taskDefinition.requiresCompatibilities,cpu: taskDefinition.cpu,memory: taskDefinition.memory}')
+
+          echo "Updating task definitions"
+          NEW_ECS_TASK_DEFINITION_API=$(echo ${ECS_TASK_DEFINITION_API} | jq '.containerDefinitions[0].image='\"${{ secrets.ecr_registry }}:${{ inputs.commit_sha }}\")
+          NEW_ECS_TASK_DEFINITION_API_MIGRATIONS=$(echo ${ECS_TASK_DEFINITION_API_MIGRATIONS} | jq '.containerDefinitions[0].image='\"${{ secrets.ecr_registry }}:${{ inputs.commit_sha }}\")
+
+          echo "Registering new task definition versions"
+          NEW_ECS_TASK_DEFINITION_API_ARN=$(aws ecs register-task-definition --region "${{ secrets.aws_region }}" --family "brave-devices-${{ inputs.sdlc }}-api" --cli-input-json "${NEW_ECS_TASK_DEFINITION_API}" | jq -r .taskDefinition.taskDefinitionArn)
+          NEW_ECS_TASK_DEFINITION_API_MIGRATIONS_ARN=$(aws ecs register-task-definition --region "${{ secrets.aws_region }}" --family "brave-devices-${{ inputs.sdlc }}-api-db-migrations" --cli-input-json "${NEW_ECS_TASK_DEFINITION_API_MIGRATIONS}" | jq -r .taskDefinition.taskDefinitionArn)
+          echo "db_migrations_task_arn=${NEW_ECS_TASK_DEFINITION_API_MIGRATIONS_ARN}" >> $GITHUB_OUTPUT
+
+      - name: Run DB Migrations
+        run: |
+          echo "Running DB Migrations"
+          API_MIGRATIONS_TASK_ARN=$(aws ecs run-task --region "${{ secrets.aws_region }}" --task-definition ${{ steps.registerTaskDefinitions.outputs.db_migrations_task_arn }} --cluster "brave-devices-${{ inputs.sdlc }}-cluster" --network-configuration "{ \"awsvpcConfiguration\": { \"assignPublicIp\":\"DISABLED\", \"securityGroups\": [\"${{ secrets.db_migrations_security_group }}\"], \"subnets\": [\"${{ secrets.db_migrations_subnet }}\"]}}" --query 'tasks[].taskArn' --output text | rev | cut -d'/' -f1 | rev)

```

Comment:
Is it possible for this step to show the output from the `setup_postgresql.sh` script? Or is the output of this script viewable somewhere else?

Right now, I just see the `echo`ed text like this:
![image](https://github.com/bravetechnologycoop/BraveSensor/assets/1490437/1218679e-272b-411f-a9f9-edaac0082da5)

Reply:
I can look into this - we would need to stream the logs from the container running the migrations, which is definitely possible. 

The logs would currently be visible in CloudWatch inside the target account, i.e. Development account in this instance.

Reply:
Need to check that if this script fails, the whole pipeline should fail.